third_party/3/jack/__init__.pyi

```diff
@@ -0,0 +1,265 @@
+from typing import Any, Callable, Dict, Generator, Iterable, Iterator, List, Optional, Sequence, Tuple, Union, overload
+
+NDArray = Any  # FIXME: no typings for numpy arrays
+
+JackPositionT = Any
```

Comment:
This is a raw C type (https://github.com/spatialaudio/jackclient-python/blob/master/src/jack.py#L220) and I'm not sure how to type it correctly so I left it as `Any`.

Reply:
Maybe type it as a private class (`class _JackPositionT: ...`).

Also, all types that don't exist at runtime should be prefixed with underscores.